server/test/controllers/ti/TrustedIntermediaryControllerTest.java

```diff
@@ -50,7 +52,7 @@ public void addApplicant_WithMissingDob() {
                         "dob",
                         "")));
     TrustedIntermediaryGroup group = repo.listTrustedIntermediaryGroups().get(0);
-    Result result = tiController.addApplicant(group.id + 1, requestBuilder.build());
+    Result result = tiController.addApplicant(group.id + 2, requestBuilder.build());
```

Comment:
Can you help me understand why this change was necessary? Is there anyway we can avoid the need for a magic number here?

Reply:
I can't :(

Reply:
this is fixed.. no magic number needed